### PR TITLE
Add Task Runner generator

### PR DESCRIPTION
**Pull request by riteshe63**
Created Initial gruntfile structure

### CODE REVIEW

File: generators/task-runner/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const _merge = require('lodash/merge');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    taskrunner() {
+      const self = this;
+
+      this.log(yosay(
+        'Configure Task Runner for Grunt'
```
Comment (gaurav21r) [inline generators/task-runner/index.js]: @riteshe63 This should be `Configure Task Runner`. We can't assume it to be Grunt at this stage

File: generators/task-runner/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      .then(answers => {
+        self.selectedTaskRunner = self.config.get('selectedTaskRunner') ||
+        answers.selectedTaskRunner;
+      });
+    },
+  },
+  writing() {
+    // Read a package.json if it exists or create it
+    const packageJSON = this.fs.readJSON(this.destinationPath('package.json'), {});
+    // Task Runner
```
Comment (gaurav21r) [inline generators/task-runner/index.js]: @riteshe63 You can remove this comment. This was for linters which had multiple of HTML, CSS, JS

File: generators/task-runner/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    if (_includes(this.selectedTaskRunner, 'Grunt')) {
+      this.log('Configuring Gruntfile for Task Runner');
+      _merge(packageJSON, {
+        devDependencies: {
+          grunt: '^1.0.1',
+          'load-grunt-config': '^0.19.2',
```
Comment (gaurav21r) [inline generators/task-runner/index.js:36]: @riteshe63 Are you sure we don't need `grunt-cli`?

File: generators/task-runner/index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        },
+      });
+      this.fs.copyTpl(
+        this.templatePath('grunt/aliases.js'),
+        this.destinationPath('./grunt/aliases.js'), {}
+      );
+      this.fs.copyTpl(
```
Comment (gaurav21r) [inline generators/task-runner/index.js:40]: @riteshe63 This should be first then aliases and how about a `this.log` with `Creating a Gruntfile` and `Creating an alias file for grunt`.

File: generators/task-runner/index.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    taskrunner() {
+      const self = this;
+
+      this.log(yosay(
+        'Configure Task Runner for Grunt'
+      ));
+      return this.prompt([{
+        type: 'checkbox',
+        name: 'selectedTaskRunner',
+        message: 'Which Task Runner do you use?',
```
Comment (gaurav21r) [inline generators/task-runner/index.js]: This should be `Which Task Runner do you want to configure?`

**Review by gaurav21r — CHANGES_REQUESTED** [review]
@riteshe63 Please note all these points. If they are done LGTM!

File: generators/task-runner/index.js (hunk 1 above)
Comment (riteshe63) [reply generators/task-runner/index.js]: Done.

File: generators/task-runner/index.js (hunk 5 above)
Comment (riteshe63) [reply generators/task-runner/index.js]: Done

File: generators/task-runner/index.js (hunk 2 above)
Comment (riteshe63) [reply generators/task-runner/index.js]: Done.

File: generators/task-runner/index.js (hunk 3 above)
Comment (riteshe63) [reply generators/task-runner/index.js:36]: Yes, I am sure.

File: generators/task-runner/index.js (hunk 4 above)
Comment (riteshe63) [reply generators/task-runner/index.js:40]: Done.

**Comment by riteshe63** [comment]
All review is done.

**Review by gaurav21r — APPROVED** [review]

**Pull request merged**